processing/data_collection/gazette/spiders/rj_belford_roxo.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime
+import re
+
+import dateparser
+from scrapy import Request
+from scrapy.exceptions import CloseSpider
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RjBelfordRoxoSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3300456"
+
+    allowed_domains = ["prefeituradebelfordroxo.rj.gov.br"]
+    name = "rj_belford_roxo"
+    start_urls = ["https://prefeituradebelfordroxo.rj.gov.br/atos-oficiais/"]
+
```

Comment:
**issue**: Add a `start_date` argument with the date of the first available gazette for this city (look `rn_natal` as reference). This will help us to limit the amount of items returned.